Documentation/api-reference.md

```diff
@@ -0,0 +1,374 @@
+     pod 'AEPEdge'
+     pod 'AEPEdgeIdentity'
+  end
+  ```
+  
+2. Install cocoapods dependencies. Replace `YOUR_TARGET_NAME` and then, in the `Podfile` directory, type:
+
+  ```bash
+  $ pod install
+  ```
+
+3. Import the Mobile Core and Edge libraries:
+
+  ```swift
+  // AppDelegate.swift
+  import AEPCore
+  import AEPEdge
+  import AEPEdgeIdentity
+  ```
+
+4. Register the Identity for Edge Extension with MobileCore:
+  ```swift
+  // AppDelegate.swift
+  func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    MobileCore.registerExtensions([Identity.self, Edge.self], {
+       MobileCore.configureWith(appId: "yourLaunchEnvironmentID")
+     })
+     ...
+  }
+  ```
+
+## API reference
+
+| APIs                                           |
+| ---------------------------------------------- |
+| [extensionVersion](#extensionVersion)          |
+| [getExperienceCloudId](#getExperienceCloudId)  |
+| [getIdentities](#getIdentities)                |
+| [getUrlVariables](#getUrlVariables)            |
+| [registerExtension](#registerExtension)        |
+| [removeIdentity](#removeIdentity)              |
+| [resetIdentities](#resetIdentities)            |
+| [updateIdentities](#updateIdentities)          |
+
+
+### extensionVersion
+
+The extensionVersion() API returns the version of the Identity for Edge Network extension.
+
+**Syntax**
+
+```swift
+static var extensionVersion: String
+```
+**Examples**
+
+```swift
+let extensionVersion = EdgeIdentity.extensionVersion
+```
+
+### getExperienceCloudId
+
+This API retrieves the Experience Cloud ID (ECID) that was generated when the app was initially launched. This ID is preserved between app upgrades, is saved and restored during the standard application backup process, and is removed at uninstall.
+
+**Syntax**
+
+```swift
+static func getExperienceCloudId(completion: @escaping (String?, Error?) -> Void)
+```
+
+* _completion_ is invoked after the ECID is available.  The default timeout is 1000ms.
+
+**Examples**
+
+```swift
+Identity.getExperienceCloudId { (ecid, error) in
+  if let error = error {
+    // handle error here
+  } else {
+    // handle the retrieved ID here
+  }
+}
+```
+
+### getIdentities
+
+Get all the identities in the Identity for Edge Network extension, including customer identifiers which were previously added.
+
+**Syntax**
+
+```swift
+static func getIdentities(completion: @escaping (IdentityMap?, Error?) -> Void)
+```
+
+* _completion_ is invoked after the identities are available.  The default timeout is 1000ms. The return format is an instance of [IdentityMap](#identitymap).
+
+**Examples**
+
+```swift
+Identity.getIdentities { (identityMap, error) in
+  if let error = error {
+    // handle error here
+  } else {
+    // handle the retrieved identities here
+  }
+}
+```
+
+### getUrlVariables
+{% hint style="info" %}
+This method was added in Edge Identity version 2.0.0.
```

Comment:
This API is available in Edge Identity version 1.1.0 and above.

Reply:
I was under the impression we should update major version when adding new API, but that is when we are not backwards compatible. In case of new API minor version bump sounds right.